invenio_vocabularies/config.py

```diff
@@ -113,6 +113,16 @@
     "subjects",
 ]
 
+VOCABULARIES_SEARCH_ATTRIBUTES = {
```

Comment:
please check my comment in https://github.com/inveniosoftware/invenio-app-rdm/pull/2776/files